tools/small_rna_map/test.r

```diff
@@ -0,0 +1,15 @@
+# Table is the data frame
+p <- ggplot(Table, aes(x=`Nbr_reads`, y=Coordinate, label=Nbr_reads)) + 
```

Comment:
to simplify the test, only use several chromosomes like this:

p <- ggplot(Table[1:27,], aes(x=`Nbr_reads`, y=Coordinate, label=Nbr_reads)) +

You will then easily see that values for both x-axis and y-axis are not good. Also, remote the black dots as well as the nbr_reads labels in these dots. the information is redundant for the moment and complicates the development